src/server.c

```diff
@@ -3254,6 +3254,18 @@ void slowlogPushCurrentCommand(client *c, struct serverCommand *cmd, ustime_t du
     slowlogPushEntryIfNeeded(c, argv, argc, duration);
 }
 
+/* Log the last command a client executed into the fatlog. */
+void fatlogPushCurrentCommand(client *c, struct serverCommand *cmd, size_t size) {
+    /* Some commands may contain sensitive data that should not be available in the fatlog. */
+    if (cmd->flags & CMD_SKIP_SLOWLOG) return;
```

Comment:
Nit about the comment: The only command marked as `SKIP_SLOWLOG` is `EXEC` (if I grep correctly), so don't think this is about sensitive data. it can be that EXEC does all the work of the commands in the transaction, so EXEC takes the time and space of these commands. It would be misleading to just report "EXEC" in the slowlog or fatlog. To make it useful, we would need to store the whole transaction in the slowlog/fatlog entry. Maybe just don't mention "sensitive data"?

Suggestion:

    /* Some commands should not be available in the slowlog and the fatlog. */

Reply:
You are right. Maybe this is originally meant for ACL commands, but they already redacted the "sensitive data".